.github/workflows/spark-rapids-jni.yaml

```diff
@@ -13,10 +13,11 @@ jobs:
         with:
           repository: NVIDIA/spark-rapids-jni
           submodules: recursive
+          ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
What if this ref doesn’t exist? We are changing branching strategies to use `main`, and sometimes have long-lived feature branches that we merge into.